typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.ts

```diff
@@ -0,0 +1,386 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Should try to honor prettier rules and only override in specific cases

Reply:
oops, good catch, thanks!

Reply:
prettier is enabled, style fixed.